src/devices/HCSR04/HCSR04.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove=".template.config\**" />
```

Comment:
One easier way of removing all of the things you don't want, is to set `EnableDefaultItems` to false and add the compile manually. Here is an example:

[https://github.com/dotnet/iot/blob/c1a3493abc77e6988e8625e2c46b3ea35eac3a91/src/devices/Mcp3008/Mcp3008.csproj#L3-L12](https://github.com/dotnet/iot/blob/c1a3493abc77e6988e8625e2c46b3ea35eac3a91/src/devices/Mcp3008/Mcp3008.csproj#L3-L12)